src/components/themed/FilledTextInput.tsx

```diff
@@ -347,6 +345,8 @@ const InnerContainer = styled(Animated.View)<{
   hasPlaceholder: boolean
 }>(theme => ({ hasPlaceholder, focusValue }) => {
   const rem = theme.rem(1)
+
+  // Need 2 pixels of shift given a 16 point rem settings
```

Comment:
Still it's weird that you need to shift the container two pixels to the right on Android. Why would Android be rendering views differently?

Reply:
Cant tell you why but that seems to be the case with text inputs vs text views

Reply:
This is an Animated.View not a TextInput

Reply:
Yes, but it's the view of the TextInput